src/Components/Box/index.js

```diff
@@ -10,11 +10,26 @@ export default class Box extends Component {
     render() {
       return (
         <div className="Rectangle-2">
-          <Picture pic={(this.props.it).name} />
-          <Name name={(this.props.it).name} />
-          <Date date={(this.props.it).height} />
-          <Contribution content={(this.props.it).mass+"sadjkandkjasndjaskndkasjndaskjjd bnkjasnbdkjasdkjasbdkaj sbddkjasjbdkja sjbdkjasbdasskj jdbaskjjdbsakj dbaskjbdkjas jbdkjasbdkja sjbdkjasbdkasd bsaskjjdbsak jdbkjasdjasjdk jasjdbasbdkja sbdkjasbdkjas bdjkasdb b"} />
+         
+          <div className="row1">
+
+              <div className="col1">
+                  <Picture pic={(this.props.it).contributor_name} />
+                  <Name name={(this.props.it).contributor_name} />
```

Comment:
why brackets.. ? (this.props.it).contributor_name can be this.props.it.contributor_name